aptos-move/aptos-gas/src/transaction.rs

```diff
@@ -65,36 +65,36 @@ crate::params::define_gas_parameters!(
         [
             gas_unit_scaling_factor: GasScalingFactor,
             "gas_unit_scaling_factor",
-            1000
+            10_000
         ],
         // Gas Parameters for reading data from storage.
-        [load_data_base: InternalGas, "load_data.base", 1],
+        [load_data_base: InternalGas, "load_data.base", 16_000],
```

Comment:
Why 16k?

Reply:
Guesswork. I just multiplied the per-byte cost by a reasonable factor. Feel free to suggest alternatives